### DISCUSSION TITLE
help flink sql中select doris表中带datetime类型的数据后BE挂掉

### DISCUSSION THREAD

**Discussion by wandayou** [apache/doris#23329] [A - General / Q&A]
BE报错日志：
*** Query id: c6b71717cdb44cd5-b294164d20aed074 ***
*** Aborted at 1692696445 (unix time) try "date -d @1692696445" if you are using GNU date ***
*** Current BE git commitID: afe6bb9 ***
*** SIGSEGV unknown detail explain (@0x0) received by PID 6843 (TID 7164 OR 0x7f6514fb6700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/src/doris/be/src/common/signal_handler.h:413
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/java/jdk1.8.0_371/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/java/jdk1.8.0_371/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/java/jdk1.8.0_371/jre/lib/amd64/server/libjvm.so
 4# 0x00007F65D9901400 in /lib64/libc.so.6
 5# doris::vectorized::DateV2Value<doris::vectorized::DateTimeV2ValueType>::to_buffer(char*, int) const at /root/src/doris/be/src/vec/runtime/vdatetime_value.cpp:2409
 6# doris::vectorized::DataTypeDateTimeV2SerDe::write_column_to_arrow(doris::vectorized::IColumn const&, unsigned char const*, arrow::ArrayBuilder*, int, int) const in /sobey/doris/be/lib/doris_be
 7# doris::vectorized::DataTypeNullableSerDe::write_column_to_arrow(doris::vectorized::IColumn const&, unsigned char const*, arrow::ArrayBuilder*, int, int) const at /root/src/doris/be/src/vec/data_types/serde/data_type_nullable_serde.cpp:120
 8# doris::FromBlockConverter::convert(std::shared_ptr<arrow::RecordBatch>*) at /root/src/doris/be/src/util/arrow/block_convertor.cpp:392
 9# doris::convert_to_arrow_batch(doris::vectorized::Block const&, std::shared_ptr<arrow::Schema> const&, arrow::MemoryPool*, std::shared_ptr<arrow::RecordBatch>*) in /sobey/doris/be/lib/doris_be
10# doris::vectorized::MemoryScratchSink::send(doris::RuntimeState*, doris::vectorized::Block*, bool) at /root/src/doris/be/src/vec/sink/vmemory_scratch_sink.cpp:83
11# doris::PlanFragmentExecutor::open_vectorized_internal() in /sobey/doris/be/lib/doris_be
12# doris::PlanFragmentExecutor::open() at /root/src/doris/be/src/runtime/plan_fragment_executor.cpp:273
13# doris::FragmentExecState::execute() at /root/src/doris/be/src/runtime/fragment_mgr.cpp:263
-------------
flink sql:
CREATE TABLE ods_contentlist_source (
		id string,
		createDate TIMESTAMP,
		personIds ARRAY<string>
    ) 
    WITH (
      'connector' = 'doris',
      'fenodes' = '172.16.152.206:8030',
      'table.identifier' = 'test.ods_contentlist',
      'username' = 'root',
      'password' = ''
);
select * from ods_contentlist_source limit 1;
--该sql执行一次则BE服务挂一次
----------------------
doris建表语句：
CREATE TABLE IF NOT EXISTS ods_contentlist
(
	id  varchar(60),
	createDate datetimev2,
	personIds ARRAY<varchar(80)> NULL
) 
UNIQUE KEY(`id`)
DISTRIBUTED BY HASH(`id`) BUCKETS 3
PROPERTIES (
"replication_allocation" = "tag.location.default: 3"
);

**Comment by zy-kkk** [comment]
please download the latest 2.0.0 https://doris.apache.org/zh-CN/download and try again
